### PR TITLE
[Term Entry] Lua Tables find() #3117

### PR DESCRIPTION
**Pull request by moha230**
### Description

- added concept entry lua tables find()
- added a folder  find
- added a file find.md
- Closes issue  #3117 

### Type of Change

- Adding a term  entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [  ] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/tables/terms/find/find.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  - 'Game Development'
+Tags:
+  - 'Tables'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **find()** is a built in function that searches for a specific value in an a table and returns its index if found or `nil` if the value is not present in the table.
```
Comment (vaibhavmani810) [inline content/lua/concepts/tables/terms/find/find.md:15]: Find() is not a built in functions in genric lua , Please refer the documentation of the Lua.

Comment (vaibhavmani810) [reply content/lua/concepts/tables/terms/find/find.md:15]: You are supposed to write the custom function find()

**Review by vaibhavmani810 — DISMISSED** [review]
Hi @moha230 , There is no built in function find() in lua language correct it accordingly also later on you did everything like custom function only but naming of your custom function is"table.find()" which is confusing , seems like inbuilt function is getting called.
Please modify these things.
After modification we will review it again.

**Comment by moha230** [comment]
Hello @vaibhavmani810 done

**Comment by SSwiniarski** [comment]
@vaibhavmani810, @moha230 This issue should have been for the Luau `table.find()` method. We shouldn't be creating custom functions for Docs, as that's not what it's for. Can we revise this PR accordingly?

**Comment by moha230** [comment]
@SSwiniarski ok will fix it Thank you

**Comment by moha230** [comment]
@vaibhavmani810 its done . Thank you

File: content/lua/concepts/tables/terms/find/find.md
Comment (SSwiniarski) [inline content/lua/concepts/tables/terms/find/find.md:1]: Can we change the location from `content/lua/concepts/tables/terms/find/find.md` to `content/luau/concepts/tables/terms/find/find.md`

File: content/lua/concepts/tables/terms/find/find.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  - 'Game Development'
+Tags:
+  - 'Tables'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **find()** function is used to search for a specific value in a table and returns its index if found.The function returns `nil` if the value is not present in the table.
```
Comment (SSwiniarski) [inline content/lua/concepts/tables/terms/find/find.md:15]:
```suggestion
The **find()** function is used to search for a specific value in a table and returns its index if found. The function returns `nil` if the value is not present in the table.
```

File: content/lua/concepts/tables/terms/find/find.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Syntax
+
+```lua
+table.find(myTable,value)
+```
+
```
Comment (SSwiniarski) [inline content/lua/concepts/tables/terms/find/find.md:22]: There should be a sentence after the syntax block explaining what `myTable` and `value` are.

File: content/lua/concepts/tables/terms/find/find.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+## Example
+
```
Comment (SSwiniarski) [inline content/lua/concepts/tables/terms/find/find.md:24]: There should be an introductory sentence summarizing what the example is doing.

File: content/lua/concepts/tables/terms/find/find.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **find()** function is used to search for a specific value in a table and returns its index if found.The function returns `nil` if the value is not present in the table.
+
+## Syntax
+
+```lua
```
Comment (SSwiniarski) [inline content/lua/concepts/tables/terms/find/find.md:19]:
````suggestion
```pseudo
````

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@moha230 I went ahead and did my own review and highlighted a few things. One of which is the file location. This was incorrectly labeled as a Lua function in the original issue when it should have been Luau. So it needs to be moved to the `luau` directory here: `content/luau/concepts/tables/terms/find/find.md`

**Comment by moha230** [comment]
@SSwiniarski  done Thank you

**Review by SSwiniarski — APPROVED** [review]
@moha230 I made a few minor changes, and now it looks good to me.

**Comment by SSwiniarski** [comment]
@vaibhavmani810 can you look this over and see if you approve merging this?

**Comment by SSwiniarski** [comment]
@vaibhavmani810 this PR needs your approval before we can merge it.

**Pull request merged**